src/MSBuild/CommandLineSwitches.cs

```diff
@@ -651,6 +656,30 @@ internal string[][] GetFileLoggerParameters()
             return groupedFileLoggerParameters;
         }
 
+        /// <summary>
+        /// Checks if the provided parametrized switch needs to be unquoted.
+        /// The method will return 'true' in case:
+        ///     The changewave 17.10 is not set and
+        ///     The parametrized switch is 'Target'
+        /// </summary>
+        private bool IsMultipleAllowedSwithParameterDueToUnquote(bool unquoteParameter, ParameterizedSwitch parameterizedSwitch)
```

Comment:
nit: IsMultipleAllowedSwithParameterDueToUnquote -> IsMultipleAllowedSwit**c**hParameterDueToUnquote

Btw. Can the name be simplified and clarified? e.g. would `ShouldUnquoteSwitchParameters` be appropriate?

Reply:
Thank you @JanKrivanek for pointing out. Applied the change: IsMultipleAllowedSwitchParameterDueToUnquote.
regarding the "ShouldUnquoteSwitchParameters" naming, I was thinking to specifically mention that the method should be applied to the multiple allowed values switch parameters, since the single valued parameters are handled differently (unquoted directly). 
It is possible also to iclude additional check so the method is actually applied to the proper parameter, what do you think?